pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/util/AuxClassPathController.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+
+package net.sourceforge.pmd.util.fxdesigner.util;
```

Comment:
I don't think this package is the right one for this... we don't have any controllers in it, and controllers are not utilities... I'd move this up to `net.sourceforge.pmd.util.fxdesigner` or maybe even create a separate package for controllers that are modals or split by screen... @oowekyala ?

Reply:
Oh yes good catch

Yeah making a new package for popups and such is a good idea.